renderer/include/Renderer.hpp

```diff
@@ -1,20 +1,45 @@
-#ifndef _Renderer_hpp_
-#define _Renderer_hpp_
+#ifndef RENDERER_HPP
+#define RENDERER_HPP
 
+#include <GL/glew.h>
+#include <SFML/Graphics.hpp>
+#include <SFML/Window.hpp>
+#include <SFML/OpenGL.hpp>
+#include <glm/glm.hpp>
+#include <glm/gtc/matrix_transform.hpp>
+#include <glm/gtc/type_ptr.hpp>
+#include <string>
 #include <GameState.hpp>
+#include "../src/Shader.hpp"
+#include "../src/Model.hpp"
+#include "../src/Camera.hpp"
 
-#include <SFML/Graphics.hpp>
+enum modelNames {
+  breakableModel, unbreakableModel, playerModel, bombModel, flameModel, balloonModel, robotModel
+};
 
-#include <vector>
+struct Model_st {
+  Model* model;
+  glm::vec3 initialPos;
+  glm::vec4 initialRot;
+  glm::vec3 initialScale;
+};
 
 class Renderer
 {
-public:
-	void render(sf::RenderWindow &window, const GameState &state);
 private:
+  std::vector<Model_st> _models;
+  Shader* _shader;
+  Camera* _camera;
 	void player(sf::RenderWindow &window, const GameState &state);
-	void enemy(sf::RenderWindow &window, const GameState &state);
 	void map(sf::RenderWindow &window, const GameState &state);
+  void enemy(sf::RenderWindow &window, const GameState &state);
+public:
+  Renderer(/* args */);
+  ~Renderer();
+  void init();
+  void render(sf::RenderWindow &window, const GameState &state);
 };
 
-#endif
+
+#endif
```

Comment:
Just for the sake of consistency, could we get a newline at the end of this file?